css/_base.scss

```diff
@@ -220,11 +220,6 @@ h1, h2, h3, h4, h5, h6 {
     cursor: pointer;
 }
 
-.jitsi-icon.gray svg {
-    fill: #5E6D7A;
-    cursor: pointer;
-}
-
```

Comment:
Stört das wenn wir das drinn lassen? Gäbe wenige Merge-Conflicte beim nächsten Update.

Reply:
In meiner Version ist das drin.